autogen/main.py

```
@@ -0,0 +1,74 @@
+
+def store_interaction(query, response, collection_name="interactions"):
+    """Almacena las interacciones en la base vectorial para aprendizaje"""
+    qdrant_client.upsert(
+        collection_name=collection_name,
+        points=[
+            {
+                "id": hash(f"{query}{response}"),
+                "payload": {
+                    "query": query,
+                    "response": response,
+                    "timestamp": datetime.now().isoformat()
```

Ruff (0.8.2)

Undefined name `datetime`

(F821)